### PR TITLE
`gpnf-populate-parent-entry-id.php`: Fixed issue where `gpnf_session_script_data_` filter callback was not returning the filtered value.

### PR DESCRIPTION
**Pull request by veryspry**
## Context

⛑️ Ticket(s): https://secure.helpscout.net/conversation/2372876723/55235/ 

## Summary

The `gpnf_session_script_data_` filter callback registered here was not returning a value which was breakin' some things.

### CODE REVIEW

File: gp-nested-forms/gpnf-populate-parent-entry-id.php [hunk 1]
```diff
@@ -21,9 +21,10 @@
 		} else {
 			// The first time the parent form loads there will be no cookie. Create a hash and set it as the default
 			// hash for this cookie created for this form.
-			$value = $session->make_hashcode();
+			$value = $session->get_runtime_hashcode();
```
Comment (veryspry) [inline gp-nested-forms/gpnf-populate-parent-entry-id.php:24]: I noticed that when we [set the default hashcode in GPNF](https://github.com/gravitywiz/gp-nested-forms/blob/53c13ffb0e10d66874d739e46f4b0a83d774a68a/includes/class-gpnf-session.php#L280), we call `get_runtime_hashcode()`. Does it make sense to do the same here? Or does it matter?

**Review by spivurno — APPROVED** [review]
Clean!

File: gp-nested-forms/gpnf-populate-parent-entry-id.php (hunk 1 above)
Comment (spivurno) [reply gp-nested-forms/gpnf-populate-parent-entry-id.php:24]: Yes! We added get_runtime_hashcode() later and it is the correct way to get the hashcode. 🙏

Comment (veryspry) [reply gp-nested-forms/gpnf-populate-parent-entry-id.php:24]: Thank you for confirming!

**Pull request merged**